package.json

```diff
@@ -54,7 +54,7 @@
     "react-tooltip": "^4.2.21",
     "redux-thunk": "^2.4.1",
     "styled-components": "^5.3.11",
-    "typescript": "3.2.1",
+    "typescript": "3.3.1",
```

Comment:
as already discussed: Can you create a user story where we bump the versions in order to reduce vulnerabilities?
Currently typescript is on major version 5.

Reply:
Here is the newly created task for typescript version upgrade:
- https://github.com/GAIA-X4PLC-AAD/portal/issues/115